rust/telemetry-sink/src/http_event_sink.rs

```diff
@@ -135,7 +144,7 @@ impl HttpEventSink {
             if let Err(e) = &result {
                 debug!("insert_stream error: {e}");
             }
-            result
+            Ok(result?)
```

Comment:
why?

Reply:
Implicit error conversion.
I need that or map_transient to return the same Result type